src/test/java/io/nats/client/impl/JetStreamManagementTests.java

```diff
@@ -400,6 +400,11 @@ private void assertInvalidConsumerUpdate(JetStreamManagement jsm, ConsumerConfig
         // 10013 consumer name already in use
         // 10105 consumer already exists and is still active
         assertTrue(e.getApiErrorCode() == 10013 || e.getApiErrorCode() == 10105);
+        assertEquals(400, e.getErrorCode());
+        assertEquals("consumer name already in use", e.getErrorDescription());
+        // \/ This is just a reminder that the server changed \/
+        // assertEquals(500, e.getErrorCode());
+        // assertEquals("consumer already exists", e.getErrorDescription());
```

Comment:
Testing server breaking change Error Message Code and Text Change

Reply:
Since it looks like you support error code, you should not have to check error strings. That is the point of the error codes is that we can change error strings (fix typos, think internationalization, etc..).

Reply:
This was just a unit test, but it's possible that a user had code like this before the release of api error codes. Also, since we cannot be sure of the version of the server used by someone, the code should be backward compatible. But this server change in specific is technically a breaking change.

Reply:
I disagree. If your library had error codes, then if user decides to check error string, they do so at their own risk and peril. As I said, the very point of having error codes is that *these* should not changes, the error text should be able to be changed (we have fixed typos for instance).